### DISCUSSION TITLE
Use statistic for gauge/sensor card

### DISCUSSION THREAD

**Discussion by balloob** [home-assistant/frontend#9734] [Dashboard]
It would be great to use a statistic as a data source for a gauge/sensor card.

It would require the following input:

 - statistic ID
 - relative start (ie start of day, 3 days ago from now)
 - relative end day (ie end of yesterday or now)

Depending on the statistic type we can either show one of min/max/mean, all of them, or if it's a meter, the growth.

Use cases:

 - energy usage yesterday
 - energy usage this month
 - energy usage last month
 - average temperature this month
 - average temperature last month

**Comment by nsmits** [comment]
I would also really like this. Personally I think the question is: how can we better use the (already present) data? (see also #9719 and #9847)
